conf/messages.en

```diff
@@ -139,6 +139,8 @@ ukRevenue.b1 = your turnover during the accounting period
 ukRevenue.b2 = any other amounts not included within your turnover that are recognised as revenue according to Generally Accepted Accounting Principles (GAAP)
 ukRevenue.b3 = if you are part of a business group, this amount is for the entity you are registering
 ukRevenue.p2 = If you have more than one relevant accounting period ending in the financial year, you need to combine all of the amounts.
+ukRevenue.p3 = If you have more than one accounting period ending in the financial year
```

Comment:
Given this is a caption/sub-heading and not plain paragraph text, I think we should reflect that in the key name, i.e. not ukRevenue.p3